test/cases/parsing/evaluate/index.js

```diff
@@ -70,3 +70,21 @@ it("should allow resourceFragment in context", function () {
 	expect(fn("index")).toEqual("#resourceFragment");
 	expect(fn("returnRF")).toBe("#..");
 });
+
+it("should try to evaluate new RegExp()", function () {
+	function expectAOnly (r) {
+		r.keys().forEach(key => {
+			expect(r(key)).toBe(1);
+		});
+	}
+
+	expectAOnly(
+		require.context("./regexp", false, new RegExp("(?<!filtered)\\.js$", ""))
+	);
+	expectAOnly(
+		require.context("./regexp", false, new RegExp(`(?<!${"filtered"})\\.js$`, "i"))
+	);
+	expectAOnly(
+		require.context("./regexp", false, new RegExp("(?<!filtered)\\.js$"))
```

Comment:
```suggestion
		require.context("./regexp", false, new RegExp("(?<!filtered)\\.js$", undefined))
```
will not work.. Did I understand right that `Identifier` `undefined` is not evaluated because technically at runtime could be
```
const undefined = 1; // in scope or
global.undefined = 1;
```

Reply:
yep, but nobody writes that code anyway...